src/renderer/pages/Staking/Overview/components/NominatorsList/ShardedList/ShardedList.tsx

```diff
@@ -93,109 +95,103 @@ export const ShardedList = ({
     });
   };
 
-  const { totalReward, totalStake } = useMemo(() => {
-    const total = shards.reduce(
-      (acc, shard) => {
-        return {
-          totalReward: acc.totalReward + +(shard.totalReward || 0),
-          totalStake: acc.totalStake + +(shard.totalStake || 0),
-        };
-      },
-      { totalReward: 0, totalStake: 0 },
-    );
+  const { totalReward, totalStake } = shards.reduce(
+    (acc, shard) => {
+      `${+acc.totalReward + +(shard.totalReward || 0)}`;
```

Comment:
`totalReward`  & `totalStake` are used in 2 places and transformed to string there
I suggest to simplify `${+acc.totalReward + +(shard.totalReward || 0)}`; as I stated earlier 
`acc.totalReward += Number(shard.totalReward || 0);`

```javascript
<AssetBalance value={totalReward.toString()} asset={asset} />
<AssetFiatBalance amount={totalReward.toString()} asset={asset} />
```